react/components/OrderInfo/index.tsx

```diff
@@ -53,11 +53,16 @@ const OrderInfo: FunctionComponent<Props> = ({
         )}
         {hasCustomerName && <CustomerInfo profile={profile} />}
         <OrderSection>
-          {paymentsData.map((payment, idx) => (
-            <div key={idx} className="flex flex-column pb8">
-              <PaymentMethod payment={payment} transactionId={transactionId} />
-            </div>
-          ))}
+          <div className="flex flex-column flex-row-m">
+            {paymentsData.map((payment, idx) => (
+              <div key={idx} className="flex flex-column pb8-s mr9-m">
```

Comment:
`flex flex-column` is useless here since `PaymentMethod` returns an `Article` element.

Reply:
Yep, removed from line 58